cypress/integration/bodySpec.js

```diff
@@ -50,4 +51,8 @@ describe('Article Body Tests', () => {
   it('should render a title', () => {
     cy.title().should('eq', "Meghan's bouquet laid on tomb of unknown warrior");
   });
+
+  it('should have working inline links', () => {
+    clickInlineLinkAndTestPageHasHTML('main a', '/news/articles/c0000000027o');
```

Comment:
We should update the description here and of the helper utility to be descriptive.